### DISCUSSION TITLE
Zowe Explorer Supported Environments

### DISCUSSION THREAD

**Discussion by JillieBeanSim** [zowe/zowe-explorer-vscode#1691] [Squad Discussion]
I think we need to discuss and document environments that we fully support.
This will help us in the future for knowing which environments we should possibly test against and help us deal with issues that may arise from users in environments that we don't usually work with and test against.

I know for sure that we test against
1. Windows
2. Mac
3. Theia
4. Ubuntu

We also keep in mind CRW with some squad members testing manually.

**Comment by zFernand0** [comment]
Maybe we can split this into categories:

- Operating Systems
  - Windows 10+
  - MacOS 10.15 (Catalina), 11 (Big Sur), 12 (Monterey)
  - *nix:
    - CentOS 8+
    - RHEL 8+
    - Ubuntu 20.04+
    
- Integrated Development Environments:
  - Theia 18+
  - [Red Hat CodeReady Workspaces](https://developers.redhat.com/products/codeready-workspaces/overview)
  - [Gitpod](https://www.gitpod-staging.com/docs/vscode-extensions/)
  - [GitHub CodeSpaces](https://github.com/features/codespaces)
  - [Eclipse Che](https://www.eclipse.org/che/docs/che-7/end-user-guide/adding-a-vs-code-extension-to-a-workspace/)
  - [VSCodium](https://vscodium.com/)
  
- Platform Versions:
  - VSCode 1.43+
  - NodeJS 14
  - Yarn 1.22.10

**Comment by JamesBauman** [comment]
@zFernand0 @JillieBeanSim 

In what article is the not supported environments documented?  I cannot locate it (in this repo).

cc: @anaxceron

**Comment by anaxceron** [comment]
https://docs.zowe.org/stable/user-guide/ze-ftp/

**Comment by JillieBeanSim** [comment]
linking the epic https://github.com/zowe/vscode-extension-for-zowe/issues/1862

**Comment by JillieBeanSim** [comment]
@anaxceron @zFernand0 have we completed all tasks for this discussion?

**Reply by zFernand0** [reply]
I believe this is done.
We have documented the environments that we support.
https://docs.zowe.org/stable/user-guide/ze-usage
